docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+-------------------
+	A :ref:`dict workflow` registered with the Dockstore GitHub App.  
+
+.. note:: This term as we define it here is associated with Dockstore and may have different definitions in other contexts.  
+
+see also :ref:`dict GitHub App registration`  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict immutable:
+
+immutable
+---------
+	Unchanging, unable to be modified. Immutability implies that an object cannot be updated.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict interoperable:
+
+interoperable
+-------------
+	The ability of data or tools from multiple resources to effectively integrate data, or operate processes, across all systems with a moderate degree of effort.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict JSON:
+
+JSON
+----
+pronounced "jason"  
+
+*abbreviation for* JavaScript Object Notation  
+
+	A human-readable file format that originated in JavaScript, but is now used by a variety of applications. Dockstore supports the inclusion of JSON and :ref:`dict YAML` files in entries to provide sample inputs for workflow and tool entries. Some workflow executors, such as :ref:`dict Cromwell`, can use these files to configure their inputs rather than having to manually listing every input when calling the workflow on the command line.  
+
+see also :ref:`dict YAML`  
+
+Further reading: `<https://www.json.org/json-en.html>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Jupyter:
+
+Jupyter
+-------
+pronounced "Jupiter" like the planet  
+
+	A project focused on developing "notebooks" for programming languages, most famously Python due to it starting as a splinter of iPython in the early 2010s, but including other languages as well such as R. Jupyter notebooks allow for blocks of code to be nestled between markdown text, allowing for easy documentation of the code blocks and reproducibility of analysis.  
+
+Further reading: `<https://jupyter.org/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict kernel:
+
+kernel
+------
+	An operating system's core program that is always loaded in memory, and modulates interactions between software and physical hardware, including but not limited to managing memory access for any program currently in RAM.  
+
+Further reading: `<https://en.wikipedia.org/wiki/Kernel_(operating_system)>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Kids First:
+
+Kids First
+----------
+*abbreviation for* Gabriella Miller Kids First Program  
+
+	An :ref:`dict NIH` program, supported by the NIH Common Fund, relating to the influence of genomics on pediatric health, with a focus on pediatric cancer and structural birth abnormalities (such as cleft palate).  
+
+Further reading: `<https://commonfund.nih.gov/kidsfirst/highlights>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict labels:
+
+labels
+------
+	On Dockstore, we use labels to "tag" Dockstore entries with information about them. You can add labels to a Dockstore :ref:`dict entry` page that you have edit access to. Labels cannot contain spaces.  
```

Comment:
Should be more specific about "you" here which is a workflow developer, distinct from categories or collections above.